src/commands/config.js

```diff
@@ -160,7 +181,7 @@ exports.handler = async ({func} = {}) => {
     }
   }
 
-  if (func) {
+  if (args.func) {
```

Comment:
This should be just `func` again, right?

Reply:
I also need to add a test for this 🤔

Reply:
There we go, tell me when the "pre-release" is done - already updated the server to develop..